internal/pipe/defaults/defaults.go

```diff
@@ -36,6 +36,7 @@ func (Pipe) Run(ctx *context.Context) error {
 		}
 
 		ctx.Config.GiteaURLs.Download = strings.ReplaceAll(apiURL, "/api/v1", "")
+		ctx.Config.GiteaURLs.Download = strings.TrimSuffix(apiURL, "/")
```

Comment:
or this entire thing could be:

```suggestion
		ctx.Config.GiteaURLs.Download = strings.TrimSuffix(strings.ReplaceAll(apiURL, "/api/v1", ""))
```

Reply:
I like this better. It makes it impossible to make the mistake I just did. Thank you!